Cargo.toml

```diff
@@ -12,13 +12,22 @@ categories = ["science"]
 keywords = ["condition", "linalg", "matrix", "ndarray", "norm"]
 
 [dependencies]
-cblas = "0.2"
-lapacke = "0.2"
-ndarray = "0.12"
+cblas = "0.4"
+lapacke = "0.5"
```

Comment:
I'd prefer to not require lapacke as well as lapack (it makes building on macOS much more difficult). Would you consider switching to using lapack? (It is mostly the same)

Reply:
lapacke was the original requirement, so I just stuck to it. Upstream, maybe I can make lapacke and lapack options to the lapack-traits package, but it's probably up to @SuperFluffy  which wrapper is used here.

I know linkage on macOS is a pain, but I got it to work out with a homebrew installed openblas backend. You just have to set the rustflags to point to the lib directories for openblas and GCC (for libgfortran). You can make your .cargo/config include something like this

```toml
[target.x86_64-apple-darwin]
rustflags = [  "-L", "native=/usr/local/opt/gcc/lib/gcc/11/",
               "-L", "native=/usr/local/opt/openblas/lib"
            ]
```

Reply:
I unfortunately have requirements to statically link all my BLAS dependencies, as I know nothing of my target system other than "macOS", so it would complicate many things to link to homebrew.

Reply:
I see the problem. I've started an upstream issue for this: hmunozb/lapack-traits#1